en/docs/guides/Provisioning/outbound-provisioning-with-salesforce.md

```diff
@@ -0,0 +1,190 @@
+# Configure Outbound Provisioning with Salesforce
+
+The WSO2 Identity Server (WSO2 IS) has the ability to provision users
+into different domains like Salesforce, Google, Facebook, etc., using
+its [identity provisioning framework](../../get-started/provisioning-architecture).
+
+This topic provides instructions on how to configure Salesforce as the
+Identity Provider to provision users from WSO2 Identity Server. The
+service provider in this scenario is WSO2 Identity Server. When WSO2 IS
+is the service provider, it is configured as the resident Service
+Provider. Therefore, after completing this tutorial you can see the
+users you add using WSO2 Identity Server being created in Salesforce
+too.
+
+## Configure Salesforce
+
+1.  Sign up as a Salesforce developer.
+    1.  Fill out the relevant information found in the following URL:
+        [https://developer.salesforce.com/signup  
+        ](https://developer.salesforce.com/signup)
```

Comment:
<https://developer.salesforce.com/signup>

enclosing the link in <> makes it a clickable link